### ISSUE TITLE
Way to identify areas in Dark Souls 3D map?

### ISSUE THREAD

**Issue by randoomdude** [magcius/noclip.website#397]
Really cool 3D map but sometimes it's hard to to tell what areas are what, maybe if you could mouse over them and have it say what area you are hovering over it would make the map a lot more clear, would be a cool thing to implement I think.

**Comment by magcius** [comment]
Have you tried using the Panels section on the left to toggle different areas?

**Comment by randoomdude** [comment]
Yeah, it doesn't work that well though since if you're unsure you have to check like 10 different panels before you know which one you're on, you could easily end up checking like 50-60 if you're checking multiple areas, just having a quick hover text would be immediate
